tasks/main.yml

```diff
@@ -3,35 +3,120 @@
 
 - include_vars: "{{ ansible_os_family }}.yml"
 
+- assert:
+    msg: dpb_cvsroot must not be empty
+    that:
+      - dpb_cvsroot is defined
+      - dpb_cvsroot | length > 0
+
+- name: Get release version
+  command: uname -r
+  register: register_release_version
+  changed_when: false
+
+- set_fact:
+    dpb_release: "{{ register_release_version.stdout }}" # x.y
```

Comment:
the fact that the command is used officially is important because it guarantees the correctness.
Although i do not understand why you would not like to leave URL, 'taken from the OpenBSD FAQ' is suffice.